### DISCUSSION TITLE
How to add the meeting list?

### DISCUSSION THREAD

**Discussion by willmunroe** [code4recovery/12-step-meeting-list#1323] [Q&A] [answered]
Hey folks, I am new to this plugin and trying to figure out how to add the actual meeting list to the site.

I can see how to preview an individual meeting but not how to add the whole list. Is there a shortcode?  Or is the page automatically generated somehow?

If the latter, how do I figure out the URL for the meeting page?

Thank you so much!

**Comment by anchovie91471** [comment, marked as answer]
It's automatically generated. yourdomain.com/meetings.

You can also find a link to it on the import & export settings page in Wordpress.

Hope this helps!

**Comment by willmunroe** [comment]
Thank you so much.  This is a great help!
